packages/client/package.json

```diff
@@ -48,14 +48,14 @@
     "License.txt"
   ],
   "dependencies": {
-    "monaco-editor": "~0.41.0",
-    "vscode": "npm:@codingame/monaco-vscode-api@1.81.7",
+    "monaco-editor": "~0.43.0",
+    "vscode": "npm:@codingame/monaco-vscode-api@1.82.0-next.0",
     "vscode-jsonrpc": "~8.1.0",
     "vscode-languageclient": "~8.1.0"
   },
   "peerDependencies": {
-    "monaco-editor": "~0.41.0",
-    "vscode": "npm:@codingame/monaco-vscode-api@1.81.7"
+    "monaco-editor": "~0.43.0",
+    "vscode": "npm:@codingame/monaco-vscode-api@1.82.0-next.0"
```

Comment:
forgot to switch on the final version?

Reply:
You must have checked an outdated version. This is `1.82.2`